spec.md

```diff
@@ -49,6 +49,135 @@ The following will not be part of the specification:
 * Language-specific runtime APIs
 * Selecting a single identity/access control system
 
+## Usage Scenarios
+
+The list below enumerates key usage scenarios and developer perspectives
+that have been considered for the development of this specification.
```

Comment:
based on March 23 WG call, I propose adding something like:

"These usage scenarios are by no means exhaustive, and the specification does not aim to be prescriptive about usage."